### PR TITLE
[TASK] Symfony command migration

### PR DESCRIPTION
**Pull request by fabarea**

### CODE REVIEW

File: Classes/Domain/Repository/BackendUserRepository.php [hunk 1]
```diff
@@ -2,69 +2,34 @@
...
     {
-        $usergroupConstraints = [];
+        $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)->getConnectionForTable('be_users')->createQueryBuilder();
+
+        $constraints = [];
         foreach ($usergroups as $usergroup) {
-            $usergroupConstraints[] = 'AND FIND_IN_SET(' . $usergroup . ', usergroup_cached_list) ';
+            $constraints[] = $queryBuilder->expr()->inSet('bu.usergroup', $queryBuilder->createNamedParameter((int)$usergroup, \PDO::PARAM_INT));
         }
-        $statement = 'SELECT * FROM be_users WHERE 1=1 ' . implode($usergroupConstraints) . BackendUtility::BEenableFields('be_users') . BackendUtility::deleteClause('be_users');
```
Comment (lorenzulrich) [inline Classes/Domain/Repository/BackendUserRepository.php:64]: I assume enableFields and deleteClause are respected automatically in the queryBuilder?

File: Classes/Service/NewsService.php [hunk 2]
```diff
@@ -36,18 +26,27 @@ class NewsService implements SingletonInterface
...
+        $q = $this->getQueryBuilder();
+        $categories = $q->select('sys_category.*')
+            ->from($this->categoryTable)
+            ->innerJoin(
+                $this->categoryTable,
+                $this->categoryMmTable,
+                $this->categoryMmTable,
+                "$this->categoryTable.uid = $this->categoryMmTable.uid_local"
+            )
+            #->innerJoin(
```
Comment (lorenzulrich) [inline Classes/Service/NewsService.php]: What is this code for?

**Comment by lorenzulrich** [comment]
Apart from the two questions, this looks OK by reading.
One question though: Does it make sense to include the complete be_users TCA or should we only include the fields we're using in the extension? What is your opinion?

File: Classes/Domain/Repository/BackendUserRepository.php (hunk 1 above)
Comment (fabarea) [reply Classes/Domain/Repository/BackendUserRepository.php:64]: Yes, that's the case. @lorenzulrich

File: Classes/Service/NewsService.php (hunk 2 above)
Comment (fabarea) [reply Classes/Service/NewsService.php]: It is to enforce / secure the mm relation, I have enabled this part again for security sake
@lorenzulrich

**Comment by fabarea** [comment]
@lorenzulrich I would say it is ok to only override the necessary part of the TCA.

**Pull request merged**